functions-and-scope.js

```diff
@@ -16,6 +16,17 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+function checkGrades(){
+    let cumLaudes = [];
+    for (let i = 0; i < grades.length; i++) {
+        if (grades[i] >= 8) {
+            cumLaudes.push(grades[i]);
+        }
+    }
+    return console.log(cumLaudes.length);
```

Comment:
console.log is een void functie. Deze returnen levert niets op. Je doet OF `return cumLaudes.length;` of `console.log(cumLaudes.length);`;